.github/workflows/test.yaml

```diff
@@ -5,5 +5,5 @@ on:
 
 jobs:
   unit-tests:
-    uses: canonical/operator-workflows/.github/workflows/test.yaml@main
```

Comment:
Do you want to merge this PR before the one on operator-workflows ?

Reply:
No, but well spotted